bibliopixel/drivers/serial_driver.py

```diff
@@ -88,12 +88,21 @@ class DriverSerial(DriverBase):
     deviceIDS = {}
     deviceVers = []
 
-    def __init__(self, type, num, dev="", c_order=ChannelOrder.RGB, SPISpeed=2, gamma=None, restart_timeout=3, deviceID=None, hardwareID="1D50:60AB"):
+    def __init__(self, type, num,
+                 dev="",
+                 c_order=ChannelOrder.RGB,
+                 SPISpeed=2,
+                 gamma=None,
+                 restart_timeout=3,
+                 deviceID=None,
+                 hardwareID="1D50:60AB",
```

Comment:
`"1D50:60AB"` ??  :-D

Yes, I know you didn't do that.  @adammhaile , what's the scoop here?